framework/include/csg/CSGPlane.h

```diff
@@ -67,9 +67,22 @@ class CSGPlane : public CSGSurface
   virtual Real evaluateSurfaceEquationAtPoint(const Point & p) const override;
 
 protected:
+  /**
+   * @brief create clone of CSGPlane object
+   *
+   * @return std::unordered_map<CSGSurface> unique_ptr to cloned plane
+   */
+  virtual std::unique_ptr<CSGSurface> clone() const override
+  {
+    return std::make_unique<CSGPlane>(_name, _a, _b, _c, _d);
+  }
+
   // calculate the equivalent coeffients (aX + bY + cZ = d) from 3 points on a plane
   void coeffsFromPoints(const Point & p1, const Point & p2, const Point & p3);
 
+  // Normalize plane coefficients so that a^2 + b^2 + c^2 = 1
```

Comment:
```suggestion
  /// Normalize plane coefficients so that a^2 + b^2 + c^2 = 1
```
// is for comments, /// is for docstring